beacon/beacon.go

```diff
@@ -137,10 +138,14 @@ type Beacon struct {
 
 // FromEvent creates Beacon request from http request parameters
 // nolint: funlen
-func FromEvent(event *types.Event) Beacon {
+func FromEvent(event *types.Event, geoIPService geoip.Service) Beacon {
 	values := event.RequestParameters
 	userAgent := event.UserAgent
 	header := event.Headers
+	country, city, err := geoIPService.CountryAndCity(header, event.RemoteAddr)
+	if err != nil {
+		log.Println(err)
```

Comment:
@ivan-stankov-golang is it possible to use throttled log here. In real life situation it's possible that we get a new range of IPs that are not available in the MaxMind DB. It's possible that we will flood our logs if we get a lot of data from such an IP range.

Also is it possible that we move 
```
country, city, err := geoIPService.CountryAndCity(header, event.RemoteAddr)
```

inside the function `ConvertToRumEvent`? I think that it makes more sense this logic to be the because need the GeoIP location for the Event object.

Reply:
I removed the log error. I move the logic to ConvertToRumEvent. Please, check and let me know what you think.